pkg/cmd/playback.go

```diff
@@ -190,24 +194,79 @@ func (pc *playbackCmd) runPlaybackCmd(cmd *cobra.Command, args []string) error {
 	select {}
 }
 
-func startListenCmdLoop(mode string, address string, httpWrapper *playback.Server) {
-	lc := newListenCmd()
-	lc.forwardURL = address + "/playback/webhooks"
-	startListenCmd := func() {
+func runListen(cmd *cobra.Command, address string) error {
+	version.CheckLatestVersion()
+
+	deviceName, err := Config.Profile.GetDeviceName()
```

Comment:
not a big deal now that we're focused on making this work and building an MVP, we'd probably want to pass something else here to clarify that the listen command is being ran through playback. I think this is used to create a CLI webhook endpoint and is shown in the dev dashboard.

Reply:
this sounds like a good clarification to add, I was thinking we could just append `-playback` to `deviceName` for now